oap-server/server-starter/src/main/resources/hierarchy-definition.yml

```diff
@@ -112,3 +115,5 @@ layer-levels:
 
   K8S_SERVICE: 0
 
+  CILIUM_SERVICE: 1
+
```

Comment:
Please order the services based on priority.

Reply:
updated.